openhabian-setup.sh

```diff
@@ -498,6 +498,29 @@ frontail() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED (service)"; exit 1; fi
 }
 
+nodered() {
+  echo -n "$(timestamp) [openHABian] Installing Node-RED... "
+  if ! command -v npm &>/dev/null; then
+    cond_redirect wget -O - https://deb.nodesource.com/setup_7.x | bash -
+    if [ $? -ne 0 ]; then echo "FAILED (prerequisites)"; exit 1; fi
+    cond_redirect apt -y install nodejs
+    if [ $? -ne 0 ]; then echo "FAILED (nodejs)"; exit 1; fi
+  fi
+  cond_redirect wget -O /tmp/update-nodejs-and-nodered.sh https://raw.githubusercontent.com/node-red/raspbian-deb-package/master/resources/update-nodejs-and-nodered || FAILED=1
+  cond_redirect bash /tmp/update-nodejs-and-nodered.sh || FAILED=1
+  if [ $FAILED -eq 1 ]; then echo "FAILED (nodered)"; exit 1; fi
+  cond_redirect npm install -g node-red-contrib-bigtimer
+  if [ $? -ne 0 ]; then echo "FAILED (nodered bigtimer addon)"; exit 1; fi
+  cond_redirect npm update -g node-red-contrib-bigtimer
+  cond_redirect npm install -g node-red-contrib-openhab2
+  if [ $? -ne 0 ]; then echo "FAILED (nodered openhab2 addon)"; exit 1; fi
+  cond_redirect npm update -g node-red-contrib-openhab2
+  cond_redirect systemctl daemon-reload
+  cond_redirect /bin/systemctl enable nodered.service
+  cond_redirect /bin/systemctl restart nodered.service
```

Comment:
You can remove `/bin/` from here. I realize that I've added this by accident over in the frontail section. Please remove there as well ;)